epix/strax_interface.py

```diff
@@ -0,0 +1,106 @@
+import epix
+import wfsim
+import numpy as np
+import pandas as pd
+import awkward1 as ak
+
+def isNumber(x):
+    try:
+        float(x)
+        return True
+    except ValueError:
+        return False
+
+@strax.takes_config(
+strax.Option('input_file',
+                    help='Input Geant4 ROOT file',track=True),
+strax.Option('entry_stop', type=int,default=None,
+                    help='Number of entries to read from first. Defaulted to all'),
+strax.Option('events_per_chunk', type=int,default=200,
+                    help='Number of events put in a chunk'),
+strax.Option('chunk_time_seperation',type=int,default=int(1e10),
+                    help='Time seperation between chunks'),
+strax.Option('micro_separation', type=float, default=0.05,track=True,
+                    help='Spatial resolution for DBSCAN micro-clustering [mm]'),
+strax.Option('micro_separation_time',type=float, default=10., track=True,
+                    help='Time resolution for DBSCAN micro-clustering [ns]'),
+strax.Option('tag_cluster_by',  type=str,default='time',track=True,
+                    help=('Classification of the type of particle of a cluster, '
+                          'based on most energetic contributor ("energy") or first '
+                          'depositing particle ("time")')),
+strax.Option('e_field', default=200,track=True,
+                    help=('Drift field map as text file ("r z E", with '
+                          'length in cm and field in V/cm) or as a constant (in V/cm; '
+                          'recommended only for testing)')),
+strax.Option('max_delay', type=float,track=True,default=1e7, #ns
+                    help='Maximal time delay to first interaction which will be stored [ns]'),
+strax.Option('timing', type=bool,default=False,
+                    help='If true will print out the time needed.'),
+strax.Option('output_path',  default="",
+                    help=('Optional output path. If not specified the result will be saved'
+                        'in the same dir as the input file.')),
+strax.Option('detector',default='XENONnT',
+                    help=('Detector which should be used. Has to be defined in epix.detectors.')),
+strax.Option('epix_config_override', type=str,
+                        default='',
+                        help='Config file to overwrite default detector settings.'),
+strax.Option('event_rate_epix', type=float,default=-1,
+                    help='Event rate for event separation. Use -1 for clean simulations'
+                         'or give a rate >0 to space events randomly.'),
+strax.Option('debug', default=False,
+                    help=('If specifed additional information is printed to the consol.')))
+class EpixInstructions(strax.Plugin):
+    '''Strax plugin providing instructions for wfsim. Needs a GEANT4 mc file as input and
+     will return the instructions for wfsim in strax format. The code in compute is pretty much
+     a copy of the run_epix script'''
```

Comment:
Docstring can be updated, since now `epix.run_epix.main` is used. -> no more code copy here.